src/main/java/com/company/TAM2/Main.java

```diff
@@ -0,0 +1,57 @@
+package com.company.TAM2;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main{
+
+    public static final int COLUMNS_COUNT = 2;
+    public static final String COMA = ",";
+    public static final int ACTION_INDEX = 0;
+    public static final int COUNT_INDEX = 1;
+
+    public static void main(String[] args) {
+        String fromFileName =  "orange.csv";
+        String toFileName = "result.txt";
+        getStatistic(fromFileName, toFileName);
+    }
+
+    public static void getStatistic(String fromFileName, String toFileName) {
+        try (
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName));
+        ) {
+            Map<String, Integer> actionCount = new HashMap<>();
+            String line;
+
+            while ((line = reader.readLine()) != null){
+                putAction(actionCount, line);
```

Comment:
according to SOLID single responsibility principe it is not good idea combine the reading from file with business logic.
- Let's method readFromFile() just reads data from file and return the string.
- Inside of getStatistic() method you process the string that was returned from previous method.
- that you create the report which is a string - reportString.
- finally you call method writeToFile(String reportString)